scripts/populate_preprint_providers.py

```diff
@@ -60,7 +59,13 @@ def set_or_update_m2m_fields(provider, emails, licenses, links, social_accounts)
         provider.licenses_acceptable.add(*licenses)
 
     for link in links:
-        obj, created = PreprintProviderLink.objects.get_or_create(**link)
+        obj, created = provider.links.update_or_create(
```

Comment:
Oh, okay. I think I was getting it confused with get_or_create which doesn't update using the defaults kwarg, I don't believe. 

I just thought that if two providers happened to create the same external link they might want the linked text to be different. It won't necessarily be unique across providers. If two providers put in the same url and description then one will update the other's linked_text.